### DISCUSSION TITLE
islandr research on vocabularies

### DISCUSSION THREAD

**Discussion by pvgenuchten** [soilwise-he/soil-vocabs#26] [Ideas]
islandr project is researching the use of common vocabularies in their research, they are happy to work with us on this topic

they created an excel with existing vocabularies

the work of islandr uses [iso11074](https://www.iso.org/obp/ui#iso:std:iso:11074:ed-2:v1:en) as a base. iso11074 is mentioned as vocabulary by many of the soil related iso standards (such as iso28258).

BRGM aims to serialize contents of iso11074 as SKOS/RDF and register it in a tool for  skos vocabularies at  [https://data.geoscience.fr/ncl/](https://data.geoscience.fr/ncl/)

**Comment by mbeaufils** [comment]
Actually @pvgenuchten the work is available here: https://data.geoscience.earth/ncl/_ISO11074

the duplication may be related to this pattern i noticed, for example the term `germination` is identified by https://data.geoscience.earth/ncl/ISO11074/_7.3.3 as well as
https://data.geoscience.earth/ncl/ISO11074/7.3.3

**Comment by pvgenuchten** [comment]
at the start of 2025, iso has released a new version of iso11074 (v2025) and brgm published the new version under (and moving the old one)

- [ISO11074v2015](https://data.geoscience.earth/ncl/_ISO11074v2015)
- [ISO11074v2025](https://data.geoscience.earth/ncl/_ISO11074v2025)

the iso visual lookups are at:
- https://www.iso.org/obp/ui#iso:std:iso:11074:ed-2:v1:en
- https://www.iso.org/obp/ui#iso:std:iso:11074:ed-3:v1:en
